projects/packages/my-jetpack/_inc/hooks/use-notification-watcher/use-get-expiring-soon-notice-content.tsx

```diff
@@ -0,0 +1,65 @@
+import { gmdateI18n } from '@wordpress/date';
+import { createInterpolateElement } from '@wordpress/element';
+import { __, sprintf } from '@wordpress/i18n';
+
+type RedBubbleAlerts = Window[ 'myJetpackInitialState' ][ 'redBubbleAlerts' ];
+
+export const useGetExpiringSoonNoticeContent = ( {
```

Comment:
It might be possible to combine these hooks as well if we just add an extra prop like `type: 'expired' | 'expiring-soon'`

Reply:
Yeah, could maybe do that, but I kinda wanted to wait to hear back from this comment: p1HpG7-umC-p2#comment-74821, to see how different and/or unique the expired notices end up being to the expiring-soon notices, once all the copy is figured out for all the different expired/expiring products.

Reply:
Actually, scratch what I said in the comment above, yes I'll combine them. 👍